packit/api.py

```diff
@@ -2497,9 +2497,9 @@ def submit_vm_image_build(
         image_name: str,
         image_request: dict,
         image_customizations: dict,
-        copr_namespace: str,
-        copr_project: str,
-        copr_chroot: str,
+        copr_namespace: Optional[str] = None,
+        copr_project: Optional[str] = None,
+        copr_chroot: Optional[str] = None,
```

Comment:
We should ideally move to the dataclass here, if and when we need more parameters, it will be a mess…